test/dump/load.txt

```diff
@@ -174,46 +174,46 @@ Code Disassembly:
 
 00001b func[0]:
  00001c: 41 00                      | i32.const 0
- 00001e: 28 02 00                   | i32.load 2 0
```

Comment:
Actually, looks at these changes now, it does seem a little odd to show the extra zero in the disassembly.  

Can we under this part with minimal code changes?   Perhaps something like:

```
if (memidx) {
  CALLBACK(OnOpcodeUint32Uint32Uint32, alignment_log2, offset, memidx);
} else {
  CALLBACK(OnOpcodeUint32Uint32, alignment_log2, offset);
}
```

Sorry for making you go back and forth on this.

Also, should we make the memidx for the first the Uint32 arguments for the load stores so it is shown first in the disassembly?

Reply:
I think it makes more sense to keep memidx as the last argument. The disassembly should be the same as the binary format which has memidx as the last argument, right?

Reply:
Sure, I don't feel strongly about it being the last argument or not, but lets completely elide it when its zero (or at least when multi memory is not enabled).

Reply:
I just updated that. Tests related to disassembly/objdump are the same as how they are in the main branch now.